Library/Client.ts

```diff
@@ -234,10 +234,11 @@ class Client {
      */
     public track(
         data: ContractsModule.Contracts.Data<ContractsModule.Contracts.Domain>,
-        tagOverrides?: { [key: string]: string; }) {
+        tagOverrides?: { [key: string]: string; },
+        contextObject?: any) {
```

Comment:
Have you looked at correlation library that will keep the context across callbacks and promises? Do you think we will need to have correlation context provided by that library passed to processors as well? Maybe `ITelemetryContext` can have a field for that context object?